pkg/networkpath/traceroute/runner.go

```diff
@@ -119,6 +120,41 @@ func (r *Runner) RunTraceroute(ctx context.Context, cfg Config) (payload.Network
 		timeout = time.Duration(cfg.TimeoutMs) * time.Millisecond
 	}
 
+	hname, err := hostname.Get(ctx)
+	if err != nil {
+		tracerouteRunnerTelemetry.failedRuns.Inc()
+		return payload.NetworkPath{}, err
+	}
+
+	protocol := strings.ToUpper(cfg.Protocol)
+	var pathResult payload.NetworkPath
+	switch protocol {
+	case TCP:
+		log.Debugf("Running TCP traceroute for: %+v", cfg)
```

Comment:
Per the profiling data, we should be careful about logging here. It consumes a lot of CPU at scale.

Reply:
Is that something that continues to consume high CPU even if log level is set to info i.e. when this becomes a noop? Or is this just a concern for when debug logging is active

Reply:
IIRC it consumes much more  when enabled, but the CPU is non-zero even  if logging is disabled. 

Also, we periodically tell customers to enable debug logging during support cases.